### PR TITLE
docs(security): proofread prose, app now shows good and bad

### PR DESCRIPTION
**Pull request by chalin**
- App now shows how Angular handles untrusted URLs and resources
- E2e test covered new functionality
- Copyedits to prose
- Updated provider expressions to use latest syntax

The original security feature tracker: https://github.com/angular/angular/issues/8511.

### CODE REVIEW

**Comment by naomiblack** [comment]
@mprobst could you do a review on this one? (and @wardbell as FYI)

File: public/docs/ts/latest/guide/security.jade [hunk 1]
```diff
@@ -139,7 +139,7 @@ figure.image-display
   ### Server side XSS protection
 
   HTML constructed on the server is vulnerable to injection attacks. Injecting template code into an
-  Angular application is the same as injecting executable code (e.g. JavaScript) into the
+  Angular application is the same as injecting executable code (e.g., JavaScript) into the
```
Comment (mprobst) [inline public/docs/ts/latest/guide/security.jade]: could also use `(that is JavaScript)`.

**Comment by mprobst** [comment]
LGTM, nice!

File: public/docs/_examples/security/ts/app/bypass-security.component.ts [hunk 2]
```diff
@@ -8,26 +8,31 @@ import { DomSanitizationService, SafeResourceUrl, SafeUrl } from '@angular/platf
...
 
   // #docregion trust-url
   constructor(private sanitizer: DomSanitizationService) {
-    // javascript: URLs are dangerous if attacker controlled. Angular sanitizes them in data
-    // binding, but we can explicitly tell Angular to trust this value:
-    this.dangerousUrl = sanitizer.bypassSecurityTrustUrl('javascript:alert("Hi there")');
+    // javascript: URLs are dangerous if attacker controlled.
+    // Angular sanitizes them in data binding, but we can
+    // explicitly tell Angular to trust this value:
+    this.dangerousUrl = 'javascript:alert("Hi there")';
```
Comment (mprobst) [inline public/docs/_examples/security/ts/app/bypass-security.component.ts]: Maybe call out in the text describing this example that Angular will log to the console (in development mode) that it sanitized an unsafe value here?

Comment (chalin) [inline public/docs/_examples/security/ts/app/bypass-security.component.ts]: Good point, will do (after this morning's standup).

File: public/docs/ts/latest/guide/security.jade (hunk 1 above)
Comment (chalin) [inline public/docs/ts/latest/guide/security.jade]: How about just dropping what is in parentheses?

**Comment by chalin** [comment]
@mprobst : changes submitted in a new "post-review" commit. 

(Note that I also cleaned up `makeExample`s, changing most of them to `makeExcerpt`).

File: public/docs/_examples/security/ts/app/bypass-security.component.ts (hunk 2 above)
Comment (chalin) [inline public/docs/_examples/security/ts/app/bypass-security.component.ts]: Done.

File: public/docs/ts/latest/guide/security.jade (hunk 1 above)
Comment (mprobst) [inline public/docs/ts/latest/guide/security.jade]: +1, it's not really adding value.

**Comment by naomiblack** [comment]
merged and live!

**Pull request closed without merging**